cmd/snap/cmd_prepare_image.go

```diff
@@ -132,6 +138,9 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 	opts.PrepareDir = x.Positional.TargetDir
 	opts.Classic = x.Classic
 
+	opts.Preseed = x.Preseed
+	opts.PreseedSignKey = x.PreseedSignKey
```

Comment:
should this error if Preseed is false but PreseedSignKey is set? or should it set Preseed implicitly? we need to do one of those things I think